### ISSUE TITLE
Update site title in proposal wizard to include current proposal name

### ISSUE THREAD

**Issue by rixx** [pretalx/pretalx#1761]
This would make it much easier to work on multiple proposals in different tabs at the same time.

We could even update the title to include a "xx time left" message during the last day of submissions.

**Comment by SkylerBurger** [comment]
Hello! I'd be happy to try adding this feature. I was able to set up the dev environment and poke around in the app, the docs are awesome by the way! I tried locating the proposal wizard mentioned in this issue's title but I don't think I found the right spot.

![Screenshot 2024-05-08 140855](https://github.com/pretalx/pretalx/assets/43080273/1c0df218-cb0f-4af9-811c-0bf648bbaeff)

In the screencap above I am logged in as the superuser and viewing a proposal. It appears the title is already updated in the tab.

![Screenshot 2024-05-08 141431](https://github.com/pretalx/pretalx/assets/43080273/648eb33f-01cd-4a00-a0a3-5a8561b952e2)

In this a screencap I'm logged in as a speaker with a proposal that has been associated to me. When viewing it, it appears the title of the site is already updated as shown in the browser tab. 

Apologies if I am looking in the completely wrong location. If this isn't the spot referred to in the issue would it be possible to include some more information regarding the page the feature should be implemented? Thank you!

**Comment by rixx** [comment]
Thanks for taking a look at this! The "proposal wizard" is the multi step form that speakers have to go to to submit their proposal in the first place – sorry, that's a common internal naming in django-formtools for multistep forms, but I should have been clearer about this!

**Comment by SkylerBurger** [comment]
I've added a PR for this issue and I'm happy to discuss any changes in scope or approach. I started with a simple JS approach before realizing this data might also be stored somewhere in a Django user session. I'm not super familiar with the user sessions but would be happy to look into it more if we want the proposal title to stay consistent in that browser tab throughout all of the remaining proposal wizard steps.
